### PR TITLE
Update commerce to latest uno.toolkit

### PR DESCRIPTION
**Pull request by jeromelaban**

### CODE REVIEW

File: src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.Shared/Styles/NavigationBar.xaml [hunk 1]
```diff
@@ -1,26 +1,25 @@
-<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-                    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-                    xmlns:utu="using:Uno.Toolkit.UI">
+<ResourceDictionary
```
Comment (Xiaoy312) [inline src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.Shared/Styles/NavigationBar.xaml:1]: might want to fix the formatting here

File: src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.Shared/MyExtensionsApp.Shared.projitems [hunk 2]
```diff
@@ -152,11 +152,31 @@
     <Content Include="$(MSBuildThisFileDirectory)appsettings.json">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
-    <Content Include="$(MSBuildThisFileDirectory)Assets\Icons\back.png" />
-    <Content Include="$(MSBuildThisFileDirectory)Assets\Icons\help_outline.png" />
-    <Content Include="$(MSBuildThisFileDirectory)Assets\Icons\close.png" />
-    <Content Include="$(MSBuildThisFileDirectory)Assets\Icons\notifications.png" />
-    <Content Include="$(MSBuildThisFileDirectory)Assets\Icons\share.png" />
+    <Content Include="$(MSBuildThisFileDirectory)Assets\Icons\back.scale-100.png" />
```
Comment (Xiaoy312) [inline src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.Shared/MyExtensionsApp.Shared.projitems:155]: should we use a wildcard here?

File: src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.Shared/Styles/NavigationBar.xaml (hunk 1 above)
Comment (jeromelaban) [reply src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.Shared/Styles/NavigationBar.xaml:1]: It's an automated copy of the commerce app, we'll have to fix it there.

File: src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.Shared/MyExtensionsApp.Shared.projitems (hunk 2 above)
Comment (jeromelaban) [reply src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.Shared/MyExtensionsApp.Shared.projitems:155]: It's a copy of the commerce app, and a shared project which is modified by VS, so it won't stay a wildcard :)

**Review by kazo0 — APPROVED** [review]

**Pull request merged**